java/vector/src/main/codegen/data/ValueVectorTypes.tdd

```diff
@@ -56,8 +56,10 @@
         { class: "Int", valueHolder: "IntHolder"},
         { class: "UInt4", valueHolder: "UInt4Holder" },
         { class: "Float4", javaType: "float" , boxedType: "Float", fields: [{name: "value", type: "float"}]},
-        { class: "IntervalYear", javaType: "int", friendlyType: "Period" }
-        { class: "Time", javaType: "int", friendlyType: "DateTime" }
+        { class: "DateDay", javaType: "int", boxedType: "Integer", friendlyType: "DateTime" },
```

Comment:
It probably is the right way but I think it overlaps with #409 and since TimeStamp already works this way, I'm inclined to keep it as is and let someone with more context clean it up at the same time as TimeStamp.